### PR TITLE
STORY-6250: rename affiliate campaign relationship endpoint

### PR DESCRIPTION
**Pull request by jebentier**
After talking with Lauren, we like this name enough that we're going to backfill this name change to all previous versions.  I'll just cherrypick this commit to other versions when this is approved.

### CODE REVIEW

File: source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Advertiser Affiliate Relationships
+Auto-Approve Affilaite to Campaigns
```
Comment (nburwell) [inline source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst]: typo in `Affiliate`

File: source/api_documentation/network_integration/index.rst [hunk 2]
```diff
@@ -11,7 +11,7 @@ Advertiser Campaigns
 Campaign Terms
 RingPools
 Promo Numbers
-Advertiser Affiliate Relationships
+Auto-Approve Affilaite to Campaigns
```
Comment (nburwell) [inline source/api_documentation/network_integration/index.rst]: typo in `Affiliate`

**Review by nburwell — CHANGES_REQUESTED** [review]
The new name is most likely the most common use case of this end point, but are we concerned that it doesn't cover the other use cases? 

By sending 'Suspended' (or 'Deactivated') you can bulk suspend all affiliate campaigns between this affiliate and advertiser.

--- 

I think we can also improve the intro paragraph to describe a bit what this does -- ie will: approve any existing affiliate campaigns (regardless of what status they were in before), and otherwise mark this affiliate as being auto-approved anytime they apply for a campaign with this advertiser in the future.

File: source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst (hunk 1 above)
Comment (jebentier) [reply source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst]: done

File: source/api_documentation/network_integration/index.rst (hunk 2 above)
Comment (jebentier) [reply source/api_documentation/network_integration/index.rst]: done

**Comment by jebentier** [comment]
Lauren and I went back and forth on that with the name and Lauren decided that the name can be this, and that the description still indicates that you can approve or deny/other actions.  I can try taking a stab at the wording for the description, but we originally talked about only the title needing to be changed

File: source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst [hunk 3]
```diff
@@ -1,9 +1,12 @@
-Advertiser Affiliate Relationships
+Auto-Approve Affiliate to Campaigns
 ==================================
 
 Changes to the Relationship between Advertiser and Affiliate on the network platform are replicated to the Invoca platform using this API.
 The operations on Advertiser‐Affiliate Relationships are similar to Network, in that the interface is fully idempotent, and the create and update commands will act as “create or update”.
 
+When setting the Relationship between an Advertiser and Affiliate to Approved, all current and future campaign applications made between the Affiliate and Advertiser will be auto approved.
```
Comment (nburwell) [inline source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst]: sorry small nit, didn't notice on slack:

You have `Auto-Approve` in the title, but use `auto approved` here in the sentence. Can we be consistent on both.

Comment (jebentier) [reply source/api_documentation/network_integration/advertiser_affiliate_relationships/index.rst]: done

**Review by nburwell — APPROVED** [review]

**Review by caitlinrhd — APPROVED** [review]
Looks good. Can you update the ticket title to reflect that we didn't end up hiding this page?

**Comment by jebentier** [comment]
done

**Pull request merged**